navigation.js

```diff
@@ -0,0 +1,25 @@
+const pageOne = document.querySelector('.book-list');
+const pageTwo = document.querySelector('.add-books');
+const pageThree = document.querySelector('.contact');
+
+const listLink = document.querySelector('.list-page');
+const addLink = document.querySelector('.add-page');
+const contactLink = document.querySelector('.contact-page');
+
+listLink.addEventListener('click', () => {
+  pageOne.style.display = 'flex';
+  pageTwo.style.display = 'none';
+  pageThree.style.display = 'none';
+});
+
+addLink.addEventListener('click', () => {
+  pageOne.style.display = 'none';
+  pageTwo.style.display = 'flex';
+  pageThree.style.display = 'none';
+});
+
+contactLink.addEventListener('click', () => {
+  pageOne.style.display = 'none';
+  pageTwo.style.display = 'none';
+  pageThree.style.display = 'flex';
+});
```

Comment:
hi there i can see you did a wonderful job adding the navigations using javascript but its not over yet. as was shown in the hint you should use the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) 

> click the link to see examples and more details

`add` and `remove` methods to display or hide content. trust me this is the easiest way  and you would be done in at list 10 lines 😄